core/src/main/java/com/amplifyframework/core/AmplifyConfiguration.java

```diff
@@ -19,8 +19,10 @@
 import android.content.res.Resources;
```

Comment:
Be sure to include `@Nullable` and `@NonNull` on every input and output of any public method in this file. Kotlin will suck, if you don't, and most of our customers use Kotlin.

Reply:
Every public method should already have the annotation.